### ISSUE TITLE
[Feature request] Allow switching off Disqus

### ISSUE THREAD

**Issue by VincentTam** [wowthemesnet/mundana-theme-jekyll#14]
Disqus has a plethora of shortcomings that deter many bloggers, say

1. privacy issues
2. slow load speed
3. no markdown support
4. poor SEO
5. high cost of migration

You may refer to these articles to know more.

1. https://victorzhou.com/blog/replacing-disqus/
1. https://old.reddit.com/r/programming/comments/b9022a/switching_off_of_disqus_reduced_my_page_weight_by/
1. https://medium.com/remys-blog/ejecting-disqus-4120e9985823

I suggest rendering Disqus only if `disqus` is not null in the site config file.

**Comment by Aspiring-Writer** [comment]
I agree, I hate Disqus

**Comment by aschrijver** [comment]
Indeed. I added it to our [Hall of Shame: Disqus "the dark commenting system" or "data machine"](https://community.humanetech.com/t/hall-of-shame-disqus-the-dark-commenting-system-or-data-machine/5164)
